packages/documentation/src/pages/getting-started/internal-tools.mdx

```diff
@@ -1,153 +1,134 @@
-   foreground with `fg` or omit the `&` character from the above command._
+*Please try to keep this list alphebetized.*
 
-1. Follow the instructions below to configure your browser.
+10. Commit and push your changes to the `devops` repo
+```bash
+git add .
+git commit -m 'add ssh keys for first-name last-name'
+git push origin add-key-first-last
+```
 
-### Accessing SOCKS proxy from the internet
+11. Create a PR for this branch on Github. In the #vetsgov-devops channel on the DSVA Slack, share a link to the PR and indicate that you need your ssh keys added to the `devops` repo.
 
-The `~/.ssh/config` file on your local system contains configuration to access the SOCKS proxy from outside the VA network (https://github.com/department-of-veterans-affairs/devops/blob/master/ssh/config#L28).
+12. Copy the SSH config from the `devops` repo to `~/.ssh`
+```bash
+cp ./ssh/config ~/.ssh/config
+```
 
-1. Run `ssh socks -D 2001 -N &`
-   _Note: the first time you connect to the jumpbox, ssh will prompt to ask
-   if you are sure you want to connect to a new host. You will be unable to
-   respond "yes" if ssh is in the background, so either bring it to the
-   foreground with `fg` or omit the `&` character from the above command._
+13. Add your SSH key to your local agent
+```bash
+ssh-add -K ~/.ssh/id_rsa_vagov
+```
+*If you're on OSX you can configure the ssh client to automatically add your keys with the by adding the following to the top of `~/.ssh/config`*
+```
+  Host *
+    AddKeysToAgent yes
+    UseKeychain yes
+    IdentityFile ~/.ssh/id_rsa
+    IdentityFile ~/.ssh/id_rsa_vagov
+```
 
-1. Follow the instructions below to configure your browser.
+14. Wait for `devops` team to review, approve, and merge your pull request
+
+*Once your pull request is merged, an update job should automatically run to update the jumpboxes and SOCKS proxy endpoints. This job takes about 5 minutes to run. Wait five minutes, then continue with* **Starting the SOCKS proxy**.
+
+### For external developers
+
+6. As part of onboarding, you should have provided your **public SSH key** to DSVA. If you haven't, do that now.
+7. DSVA will add your public SSH key to the list of authorized keys.
+8. When that process is completed (24 hours), you will receive a Github notification that your key has been added.
+
+*Once you receive the notification, continue with* **Starting the SOCKS proxy**.
+
+## Starting the SOCKS proxy
+There are slightly different commands to connect to the proxy depending on whether you are connected to the VA network or not. The `~/.ssh/config` file on your local system contains configuration to access the SOCKS proxy both from the VA network and the internet.
+
+_The first time you connect to the jumpbox, ssh will prompt to ask if you are sure you want to connect to a new host. You will be unable to respond "yes" if ssh is in the background, so either bring it to the foreground with `fg` or omit the `&` character from the above command._
```

Comment:
[ ] TODO: reorder the content in this section so that the "first time" instructions appear after the commands.